.changeset/two-suns-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```

Comment:
imo, this could just be a patch. It's essentially just a bug fix. It's also unlikely we'll be doing another minor before 5.0.

Reply:
yep, agreed. changed to a patch